### PR TITLE
ci: trigger homebrew update script

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Update the release workflow to trigger homebrew formula job

Signed-off-by: Darren Murray <darren.murray@lacework.net>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -138,6 +138,10 @@ jobs:
           fail_only: true
           mentions: << pipeline.parameters.slack-mentions >>
           only_for_branches: <<pipeline.parameters.only_for_branches>>
+  trigger-homebrew-update:
+    executor: alpine
+    steps:
+      - run: curl -u "$CIRCLE_TOKEN" https://circleci.com/api/v1.1/project/gh/lacework/homebrew-tap/tree/main -d "build_parameters[CIRCLE_JOB]=update"
```
Comment (afiune) [inline .circleci/config.yml:144]: Really quick, why we don't use API v2? 🤔 https://circleci.com/docs/api/v2/#operation/triggerPipeline

Comment (dmurray-lacework) [reply .circleci/config.yml:144]: I don't see the option to trigger an individual job on the v2 api. As far as I know only the 1.1 api allows specifying a job. In this case we only want to run update.

Comment (afiune) [reply .circleci/config.yml:144]: Understood. I wonder if we should implement this trigger using workflows and parameters instead 🤔 

- We can revisit this later, let's move forward with this implementation.

**Review by afiune — APPROVED** [review]
tenor-130489947

File: .circleci/config.yml (hunk 1 above)
Comment (dmurray-lacework) [reply .circleci/config.yml:144]: Sounds Good!

**Pull request merged**